src/engine/main/NetworkManager.C

```diff
@@ -5006,10 +5011,18 @@ NetworkManager::ExportSingleDatabase(int id, const ExportDBAttributes &atts)
 
         std::string qualFilename;
         if (atts.GetDirname() == "")
+        {
             qualFilename = atts.GetFilename();
+        }
         else
-            qualFilename = atts.GetDirname() + std::string(VISIT_SLASH_STRING)
-                         + atts.GetFilename();
+        {
+            // use std::filesystem::path to help us make sure we are using
+            // native style paths
+            auto dname = std::filesystem::path(atts.GetDirname());
+            auto fname = std::filesystem::path(atts.GetFilename());
+            // Cyrus Note: I hate this `/` operator nonsense, why not a join method?
+            qualFilename = (dname / fname).native();
```

Comment:
```suggestion
            qualFilename = (dname / fname).make_preferred().string();
```
I tested the suggestion on Windows and it works. Should verify the same on Mac/Linux.